apps/src/code-studio/progressRedux.js

```diff
@@ -365,9 +365,7 @@ const userProgressFromServer = (state, dispatch, userId = null) => {
       );
     }
 
-    if (data.lockableAuthorized) {
-      dispatch(authorizeLockable());
-    }
+    dispatch(authorizeLockable(data.lockableAuthorized));
```

Comment:
This might be related to my other questions but what happens now if data.lockableAuthorized is null?  When does this case occur?

Reply:
It's null before the data is loaded, instead of defaulting to false.